packages/lit-html/src/lit-html.ts

```diff
@@ -609,9 +619,24 @@ export interface RenderOptions {
 
 /**
  * Renders a value, usually a lit-html TemplateResult, to the container.
- * @param value
- * @param container
- * @param options
+ *
+ * For example, to render "Hello, Zoe!" to the container `document.body`:
```

Comment:
This example doesn't differentiate the string "Hello, Zoe!" from the HTML content "<p>Hello, Zoe</p>". Maybe there's a way to phrase it so that it does? It also doesn't mention that there's a variable in the example - which might be good or might be noise, I'm not sure.

Reply:
Removed variable, but unsure how to address the differentiating between HTML content.

I added a link to the relevant lit.dev in a `@see` tag at the end.

Reply:
I think keeping the variable and expression is valuable, especially if our default example uses `html`.

How about
```
For example, to render "Hello, Zoe!" to the container `document.body` where the name "Zoe" is from a variable:
```